### PR TITLE
[CCXDEV-13025] Add integration tests for io-gathering-service

### PR DESCRIPTION
**Pull request by matysek**
Attempt to replace pr_check for io-gathering-service

### CODE REVIEW

File: cluster/stone-prd-rh01/tenants/obsint-processing-tenant/io-gathering-service-bonfire-tekton.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    - name: IQE_SELENIUM
+      value: # Whether to run IQE pod with a selenium container. Default is "false"
+    - name: IQE_PARALLEL_ENABLED
+      value: # Whether to run IQE in parallel mode. Default is "false"
+    - name: IQE_PARALLEL_WORKER_COUNT
+      value: # The number of parallel workers to use. Default is "".
+    - name: IQE_RP_ARGS
+      value: # Arguments to send to reportportal. Default is "".
+    - name: IQE_IBUTSU_SOURCE
+      value: # Ibutsu source for the current run. Default is "".    - name: APP_NAME
```
Comment (Bee-lee) [inline cluster/stone-prd-rh01/tenants/obsint-processing-tenant/io-gathering-service-bonfire-tekton.yaml:62]: seems to be a leftover `- name: APP_NAME`?

Comment (matysek) [reply cluster/stone-prd-rh01/tenants/obsint-processing-tenant/io-gathering-service-bonfire-tekton.yaml:62]: Probably not: for konflux we have a separate app name: `io-gathering-service`. For app-interface we use `ccx-data-pipeline`.

Comment (juandspy) [reply cluster/stone-prd-rh01/tenants/obsint-processing-tenant/io-gathering-service-bonfire-tekton.yaml:62]: @Bee-lee refers to the `    - name: APP_NAME` part in the end of the line

File: cluster/stone-prd-rh01/tenants/obsint-processing-tenant/io-gathering-service-bonfire-tekton.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  namespace: obsint-processing-tenant
+spec:
+  application: io-gathering-service
+  contexts:
+    - description: Application testing
+      name: application
+  resolverRef:
+    params:
+      - name: url
+        value: https://github.com/RedHatInsights/bonfire-tekton.git # Temporary on gbenhaim's org. Also, you can fork it and reference yours here.
```
Comment (juandspy) [inline cluster/stone-prd-rh01/tenants/obsint-processing-tenant/io-gathering-service-bonfire-tekton.yaml:16]: > `Temporary on gbenhaim's org`

this is not true, right (?)

File: cluster/stone-prd-rh01/tenants/obsint-processing-tenant/io-gathering-service-bonfire-tekton.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
+apiVersion: appstudio.redhat.com/v1beta1
+kind: IntegrationTestScenario
+metadata:
+  labels:
+    test.appstudio.openshift.io/optional: "true" # Change to "true" if you don't need the test to be mandatory
```
Comment (juandspy) [inline cluster/stone-prd-rh01/tenants/obsint-processing-tenant/io-gathering-service-bonfire-tekton.yaml:5]: why is it optional?

File: cluster/stone-prd-rh01/tenants/obsint-processing-tenant/io-gathering-service-bonfire-tekton.yaml (hunk 2 above)
Comment (matysek) [reply cluster/stone-prd-rh01/tenants/obsint-processing-tenant/io-gathering-service-bonfire-tekton.yaml]: fixed. thanks

File: cluster/stone-prd-rh01/tenants/obsint-processing-tenant/io-gathering-service-bonfire-tekton.yaml (hunk 3 above)
Comment (matysek) [reply cluster/stone-prd-rh01/tenants/obsint-processing-tenant/io-gathering-service-bonfire-tekton.yaml]: fixed.

**Review by Bee-lee — APPROVED** [review]

**Review by ernesgonzalez33 — APPROVED** [review]

**Pull request merged**